pylint/checkers/refactoring.py

```diff
@@ -57,6 +57,7 @@ def _is_node_return_ended(node):
             return False
         exc_name = exc.pytype().split('.')[-1]
         handlers = utils.get_exception_handlers(node, exc_name)
+        handlers = list(handlers) if handlers is not None else list()
```

Comment:
Not sure what pylint's coding style is here, but usually `[]` is more idiomatic than `list()`.

Reply:
Ok! Thanks for the remark.